data/mods/Magiclysm/scenarios.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "type": "scenario",
+    "ident": "lost_faith",
+    "name": "Lost Faith",
+    "flags": [ "SUR_START", "LONE_START" ],
+    "points": 2,
+    "description": "You've been preaching your faith during a long time, but the recent events make you doubt about it. Wandering in your temple, you just found books that may restore your faith or finish to shatter it.",
```

Comment:
I don't like the wording here. "finish to shatter it" is very clunky.

Reply:
"erode it even more" ?

Reply:
"shatter it completely"?